modules/ROOT/pages/salesforce-to-twilio.adoc

```diff
@@ -0,0 +1,454 @@
+
+There is a good number of core components and there are many connectors. For this tutorial, you'll use these:
+
+* Salesforce connector
+* Set Variable component
+* Choice router component
+* Database connector
+* Set Payload component
+* Transform component
+* Twilio connector
+
+For this project, you'll create only a single flow.
+
+Follow these steps to create the project.
+
+. Log into Anypoint Platform.
+. Open Design Center and create a new Mule application.
+.. Click *Create* next to the search field.
+.. In the *Create a Mule Application* section of the overlay, click *Create New Application*.
+.. Name your project `Order-to-cash automation`.
+. In the *Let’s Get Started* wizard, click the link `Go straight to canvas`, which is in the bottom-left corner.
++
+*Result:* Flow Designer appears, an empty trigger card displayed in the middle of the canvas.
++
+image::canvas-empty-trigger.png[]
+
+== Configure the trigger for the flow
+
+A trigger is an event that starts a flow in a Mule application. In your project, the trigger is the change in status of an opportunity in Salesforce Sales Cloud to `Closed Won`. You can configure the trigger card to represent a Salesforce connector that is subscribed to the topic `ClosedOpps`, which you created above in "Capture opportunities that are won".
+
+. Click the Trigger card.
++
+*Result:* A list of the available connectors is displayed in the card.
+. In the *Search* field, type “Salesforce”.
++
+*Result:* The Salesforce connector appears in the list.
+. Select the Salesforce connector.
++
+*Result:* A list of operations that you can use to trigger Mule applications with the Salesforce connector is displayed.
+. Select the operation *Subscribe Topic*.
++
+*Result:* The card is opened. In its top-left corner, it is identified as a Salesforce connector. The name of the operation -- Subscribe Topic -- is also displayed. At the top of the card is the message “You need to set up the Salesforce configuration. Click here to set it up.”
+. Specify how you want Flow Designer to authenticate to Salesforce.
+.. Click the linked text that is in the message.
+.. In the *Connection Type* field, select *Username Password (Deprecated)*.
+.. Specify your username, password, and security token.
+.. Click the *Test* button to check whether Flow Designer can connect with the credentials that you specified. If the test fails, double-check your credentials and token. Save the configuration after the test succeeds.
+. Add your Push Topic to the connector by typing `ClosedOpps` in the *Topic* field. Then, close the card either by clicking the X in the top-right corner or by clicking an empty space on the canvas.
+
+== Configure the card for Twilio
+
+Because you want to notify the customer with an SMS, you can a Twilio connector and its Send Message operation.
```

Comment:
@rddheath Please change 
'Because you want to notify the customer with an SMS, you can a Twilio connector and its Send Message operation.'
to
'Because you want to notify the customer with an SMS, you can use the Twilio connector and its Send Message operation.'